src/rules/at-rule-blockless-newline-after/index.js

```diff
@@ -0,0 +1,60 @@
+import {
+  cssStatementHasBlock,
+  rawNodeString,
+  report,
+  ruleMessages,
+  validateOptions,
+  whitespaceChecker,
+} from "../../utils"
+
+export const ruleName = "at-rule-blockless-newline-after"
+
+export const messages = ruleMessages(ruleName, {
+  expectedAfter: () => "Expected newline after blockless at-rule",
+})
+
+export default function (actual) {
+  const checker = whitespaceChecker("newline", actual, messages)
+
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual,
+      possible: ["always"],
+    })
+    if (!validOptions) { return }
+
+    root.walkAtRules(statement => {
```

Comment:
You might want to name this param `atRule` rather than `statement`. We tend to use `statement` when a function is checking both `rules` and `@rules`, as it helps us distinguish between the scope of a rule.